nipap-www/nipapwww/public/controllers.js

```diff
@@ -9,21 +9,119 @@ var nipapAppControllers = angular.module('nipapApp.controllers', []);
  */
 nipapAppControllers.controller('VRFListController', function ($scope, $http) {
 
-	// Fetch VRFs from backend
-	$http.post('/xhr/list_vrf',
-		JSON.stringify({}),
-		{ 'headers': { 'Content-Type': 'application/json' } })
-		.then(function (response) {
-			if (response.data.hasOwnProperty('error')) {
-				showDialogNotice('Error', response.data.message);
-			} else {
-				$scope.vrfs = response.data;
-			}
-		})
-		.catch(function (response) {
-			var msg = response.data || "Unknown failure";
-			showDialogNotice('Error', response.status + ': ' + msg);
-		});
+	var isSmartSearch;
```

Comment:
Can't see this extra indent in my code. It should be a new blank row below "nipapAppControllers.controller('VRFListController', function ($scope, $http) {"

![image](https://user-images.githubusercontent.com/70938081/99264564-e9b5c380-2820-11eb-8b09-4727dec0347f.png)

I tried to follow how the following method was written:
 "/*
 * PoolListController - used to list pools on the /pool/list-page
 */
nipapAppControllers.controller('PoolListController', function ($scope, $http) {"
![image](https://user-images.githubusercontent.com/70938081/99264658-0ce07300-2821-11eb-89a8-22a951d3f8cb.png)